webapp-django/crashstats/topcrashers/jinja2/topcrashers/topcrashers.html

```diff
@@ -164,18 +164,39 @@
+                                {% if crash.startup_count == crash.count %}
+                                <img src="{{ static('img/3rdparty/silk/flag_red.png') }}"
+                                    alt="red flag"
+                                    title="Startup Crash, all crashes happened during startup"
+                                    class="startup" height="16" width="16"
+                                />
+                                {% else %}
+                                <img src="{{ static('img/3rdparty/silk/flag_yellow.png') }}"
+                                    alt="yellow flag"
+                                    title="Potential Startup Crash, {{ crash.startup_count }} out of {{ crash.count }} crashes happened during startup"
+                                    class="startup" height="16" width="16"
+                                />
+                                {% endif %}
+                                {% endif %}
+
                                 {% if crash.startup_crash %}
-                                <img src="{{ static('img/icons/rocket_fly.png') }}" alt="Startup Crash" title="Startup Crash" class="startup" height="16" width="16">
+                                <img
+                                    src="{{ static('img/icons/rocket_fly.png') }}"
+                                    alt="rocket"
+                                    title="Potential Startup Crash, more than half of the crashes happened during the first minute after launch"
+                                    class="startup" height="16" width="16"
+                                />
                                 {% endif %}
 
                                 {% if crash.hang_count > 0 %}
-                                <img src="{{ static('img/3rdparty/fatcow/stop16x16.png') }}" alt="Hanged Crash" title="Hanged Crash" class="hang" height="16" width="16">
+                                <img src="{{ static('img/3rdparty/fatcow/stop16x16.png') }}" alt="stop sign" title="Hanged Crash" class="hang" height="16" width="16">
                                 {% endif %}
 
                                 {% if crash.plugin_count > 0 %}
-                                <img src="{{ static('img/3rdparty/fatcow/brick16x16.png') }}" alt="Plugin Crash" title="Plugin Crash" class="plugin" height="16" width="16">
+                                <img src="{{ static('img/3rdparty/fatcow/brick16x16.png') }}" alt="brick" title="Plugin Crash" class="plugin" height="16" width="16">
                                 {% else %}
-                                <img src="{{ static('img/3rdparty/fatcow/application16x16.png') }}" width="16" height="16" alt="Browser Crash" title="Browser Crash" class="browser" />
+                                <img src="{{ static('img/3rdparty/fatcow/application16x16.png') }}" width="16" height="16" alt="application" title="Browser Crash" class="browser" />
```

Comment:
But it is needed in JSX. I'm used to having them and I don't like having to switch between writing that ``/`` or not depending on the context, so I just put them everywhere. Also, that's perfectly valid HTML.